Sources/ProjectSpec/TargetSource.swift

```diff
@@ -128,10 +128,6 @@ extension TargetSource: JSONEncodable {
             dict["optional"] = optional
         }
 
-        if dict.count == 0 {
-            return path
-        }
-
```

Comment:
In this case we can also move `dict["path"] = path` up above

Reply:
Thanks for your kind suggestions!

I reflected it in the commit below.  Please check it ;)
https://github.com/yonaskolb/XcodeGen/pull/928/commits/72215cdba669240e92a1956859444e683f8e6e05